docker-compose.litecoind.yml

```diff
@@ -0,0 +1,45 @@
+##########################################
+#
+# sparkswap Broker-CLI and Broker-Daemon
+# https://sparkswap.com
+#
+# The broker and engines currently default to TestNet. To adjust settings, please
+# look at the associated `.env` files
+#
+# Troubleshooting GRPC:
+# - GRPC_VERBOSITY=INFO
+# - GRPC_TRACE=all
+#
+##########################################
+
+version: '2.4'
+
+services:
+  lnd_ltc:
+    depends_on:
+      - litecoind
+
+  litecoind:
+    image: sparkswap_litecoind:latest
+    volumes:
+      - shared:/shared
+      - litecoin:/data
+    environment:
+      - RPC_USER=${LTC_RPC_USER}
+      - RPC_PASS=${LTC_RPC_PASS}
+      - NETWORK=${NETWORK}
+      - DEBUG=info
+      - DATA_DIR=/data
+      - RPC_LISTEN=0.0.0.0
+    networks:
+      - broker
+    logging:
+      options:
+        max-size: 50m
+
+volumes:
+  shared:
```

Comment:
Is it ok to declare this volume twice like this? Is it the same volume that's declared in the other file?

Reply:
Yes, its fine. It just merges all of the volumes together. We also could remove it in all but docker-compose.yml, but I think it is confusing to have volumes defined above and then not have them below. It also makes it so you could never independently run the specific docker-compose file. 

https://docs.docker.com/compose/extends/#adding-and-overriding-configuration